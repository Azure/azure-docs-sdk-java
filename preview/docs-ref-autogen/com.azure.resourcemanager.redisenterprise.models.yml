### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.redisenterprise.models"
fullName: "com.azure.resourcemanager.redisenterprise.models"
name: "com.azure.resourcemanager.redisenterprise.models"
summary: "Package containing the data models for RedisEnterpriseManagementClient. REST API for managing Redis Enterprise\n resources in Azure."
classes:
- "com.azure.resourcemanager.redisenterprise.models.ActionType"
- "com.azure.resourcemanager.redisenterprise.models.AofFrequency"
- "com.azure.resourcemanager.redisenterprise.models.ClusterList"
- "com.azure.resourcemanager.redisenterprise.models.ClusterUpdate"
- "com.azure.resourcemanager.redisenterprise.models.ClusteringPolicy"
- "com.azure.resourcemanager.redisenterprise.models.DatabaseList"
- "com.azure.resourcemanager.redisenterprise.models.DatabaseUpdate"
- "com.azure.resourcemanager.redisenterprise.models.ErrorAdditionalInfo"
- "com.azure.resourcemanager.redisenterprise.models.EvictionPolicy"
- "com.azure.resourcemanager.redisenterprise.models.ExportClusterParameters"
- "com.azure.resourcemanager.redisenterprise.models.ImportClusterParameters"
- "com.azure.resourcemanager.redisenterprise.models.Module"
- "com.azure.resourcemanager.redisenterprise.models.OperationDisplay"
- "com.azure.resourcemanager.redisenterprise.models.OperationListResult"
- "com.azure.resourcemanager.redisenterprise.models.Origin"
- "com.azure.resourcemanager.redisenterprise.models.Persistence"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpoint"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.redisenterprise.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.redisenterprise.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.redisenterprise.models.Protocol"
- "com.azure.resourcemanager.redisenterprise.models.ProvisioningState"
- "com.azure.resourcemanager.redisenterprise.models.RdbFrequency"
- "com.azure.resourcemanager.redisenterprise.models.RegenerateKeyParameters"
- "com.azure.resourcemanager.redisenterprise.models.ResourceState"
- "com.azure.resourcemanager.redisenterprise.models.Sku"
- "com.azure.resourcemanager.redisenterprise.models.SkuName"
- "com.azure.resourcemanager.redisenterprise.models.TlsVersion"
enums:
- "com.azure.resourcemanager.redisenterprise.models.AccessKeyType"
interfaces:
- "com.azure.resourcemanager.redisenterprise.models.AccessKeys"
- "com.azure.resourcemanager.redisenterprise.models.Cluster"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.Definition"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.Blank"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithMinimumTlsVersion"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithSku"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithTags"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.DefinitionStages.WithZones"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.Update"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.UpdateStages"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.UpdateStages.WithMinimumTlsVersion"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.UpdateStages.WithSku"
- "com.azure.resourcemanager.redisenterprise.models.Cluster.UpdateStages.WithTags"
- "com.azure.resourcemanager.redisenterprise.models.Database"
- "com.azure.resourcemanager.redisenterprise.models.Database.Definition"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.Blank"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithClientProtocol"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithClusteringPolicy"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithEvictionPolicy"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithModules"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithPersistence"
- "com.azure.resourcemanager.redisenterprise.models.Database.DefinitionStages.WithPort"
- "com.azure.resourcemanager.redisenterprise.models.Database.Update"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages.WithClientProtocol"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages.WithClusteringPolicy"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages.WithEvictionPolicy"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages.WithModules"
- "com.azure.resourcemanager.redisenterprise.models.Database.UpdateStages.WithPersistence"
- "com.azure.resourcemanager.redisenterprise.models.Databases"
- "com.azure.resourcemanager.redisenterprise.models.Operation"
- "com.azure.resourcemanager.redisenterprise.models.OperationStatus"
- "com.azure.resourcemanager.redisenterprise.models.Operations"
- "com.azure.resourcemanager.redisenterprise.models.OperationsStatus"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.UpdateStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnection.UpdateStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.redisenterprise.models.PrivateLinkResource"
- "com.azure.resourcemanager.redisenterprise.models.PrivateLinkResources"
- "com.azure.resourcemanager.redisenterprise.models.RedisEnterprises"
metadata: {}
package: "com.azure.resourcemanager.redisenterprise.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-redisenterprise:1.0.0-beta.2
