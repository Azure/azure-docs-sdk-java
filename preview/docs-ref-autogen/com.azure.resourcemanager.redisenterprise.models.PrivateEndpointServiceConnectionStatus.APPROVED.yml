### YamlMime:JavaMember
uid: "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.redisenterprise.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.redisenterprise.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-redisenterprise:1.0.0-beta.2
